uptycs/README.md

```diff
@@ -0,0 +1,85 @@
+# Uptycs
+
+## Overview
+
+Your developer's laptop is just a hop away from cloud infrastructure. Attackers don't think in isolated systems, so why would you rely on fragmented solutions to protect public cloud, private cloud, containers, laptops, and servers? Uptycs mitigates risk by prioritizing your responses to threats, vulnerabilities, misconfigurations, sensitive data exposure, and compliance requirements across your modern attack surface—all accessible through a single user interface and data model. This includes the capability to correlate threat activity as it traverses on-prem and cloud boundaries, providing a more comprehensive enterprise-wide security posture. Looking for acronym coverage? We've got you covered with CNAPP, CWPP, CSPM, KSPM, CIEM, CDR, and XDR. Start with your Detection Cloud, utilize Google-like search, and the attack surface coverage you need today. Stay prepared for what lies ahead.
```

Comment:
```suggestion
Uptycs mitigates risk by prioritizing your responses to threats, vulnerabilities, misconfigurations, sensitive data exposure, and compliance requirements across your modern attack surface — all accessible through a single user interface and data model. This includes the capability to correlate threat activity as it traverses on-prem and cloud boundaries, providing a more comprehensive enterprise-wide security posture. Looking for acronym coverage? We've got you covered with CNAPP, CWPP, CSPM, KSPM, CIEM, CDR, and XDR. Start with your Detection Cloud, utilize Google-like search, and the attack surface coverage you need today. Stay prepared for what lies ahead.
```
We try to avoid marketing-style language on this repository, so I recommend removing this part.

Reply:
updated as suggested